etc/config-template.toml

```diff
@@ -927,6 +927,12 @@
 ## default: lz4
 # blob-file-compression = "lz4"
 
+## Set blob file zstd dictionary compression, default(0) will use zstd compression
+## It is recommend to set dictionary size to 4k, 16k, and sample data to train 
+## dictionary is of size 100X dictionary size.
+## It has no effect when `blob-file-compression` is not `zstd`.
+# zstd-dict-size = 0
```

Comment:
Why the default value is 0 but the recommended size is 4K or 16K.

Reply:
It means when enabling the dictionary compression, recommend to set size to 4k/16k